pkg/curatedpackages/package.go

```diff
@@ -0,0 +1,55 @@
+package curatedpackages
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"text/tabwriter"
+
+	api "github.com/aws/eks-anywhere-packages/api/v1alpha1"
+)
+
+const (
+	MinWidth = 16
+	TabWidth = 8
+	Padding  = 0
+	PadChar  = '\t'
+	flags    = 0
+)
+
+func DisplayPackages(packages []api.BundlePackage) {
+	m := packagesToString(packages)
+
+	// initialize tabwriter
+	w := new(tabwriter.Writer)
+
+	w.Init(os.Stdout, MinWidth, TabWidth, Padding, PadChar, flags)
+	defer w.Flush()
+
+	fmt.Fprintf(w, "\n %s\t%s\t", "Package", "Version(s)")
+	fmt.Fprintf(w, "\n %s\t%s\t", "----", "----")
+	for key, values := range m {
+		fmt.Fprintf(w, "\n %s\t%s\t", key, strings.Join(values, ","))
+	}
+}
```

Comment:
```suggestion
func DisplayPackages(packages []api.BundlePackage) {
	w := new(tabwriter.Writer)
	w.Init(os.Stdout, MinWidth, TabWidth, Padding, PadChar, flags)
	defer w.Flush()

	fmt.Fprintf(w, "\n %s\t%s\t", "Package", "Version(s)")
	fmt.Fprintf(w, "\n %s\t%s\t", "----", "----")
	for _, pkg := range packages {
		versions := convertBundleVersionToPackageVersion(pkg.Source.Versions)
		fmt.Fprintf(w, "\n %s\t%s\t", pkg.Name, strings.Join(versions, ","))
	}
}
```

Reply:
How extensively has this been tested? Trying to create alignment can be a tricky problem particularly when you have variable length strings. I.e. are we confident `pkg.Name` is always going to be short enough to fit under `Package\t`? There are packages dedicated to this kind of formatting problem.

Reply:
By any chance, could you recommend the package that solves this kind of formatting problem?

Reply:
I don't recall what I've used before but a quick search shows https://github.com/buger/goterm which seems well supported.